bin/hashbangctl

```diff
@@ -109,13 +112,13 @@ while True:
 
     key = getch()
 
-    if key in [ key for k,v in menu_items ]:
+    if key in [ key for k, v in menu_items ]:
         print("%s\n" % key)
 
-    if ord(key) == 3: # <Ctrl+C>
-        break;
+    if ord(key) == 3:  # <Ctrl+C>
```

Comment:
@ChickenNuggers It's a PEP8 recommendation, and I agree it improves readability.